game/songs.cc

```diff
@@ -1,28 +1,29 @@
 #include "songs.hh"
 
 #include "configuration.hh"
-#include "fs.hh"
-#include "song.hh"
 #include "database.hh"
+#include "fs.hh"
 #include "i18n.hh"
-#include "profiler.hh"
+#include "json.hh"
 #include "libxml++-impl.hh"
-#include "unicode.hh"
+#include "log.hh"
 #include "platform.hh"
+#include "profiler.hh"
+#include "song.hh"
+#include "unicode.hh"
 
 #include <algorithm>
 #include <cstddef>
 #include <cstdlib>
 #include <iostream>
+#include <fstream>
 #include <regex>
 #include <stdexcept>
 
-#include "fs.hh"
 #include <fmt/format.h>
+#include <nlohmann/json.hpp>
 #include <unicode/stsearch.h>
```

Comment:
Change this order to be:
1. local headers
2. 3rd party headers
3. std headers